decidim-core/app/views/layouts/decidim/_logo.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if organization %>
-  <%= link_to decidim.root_url(host: organization.host), title: t("logo", scope: "decidim.accessibility", organization: organization.name) do %>
+  <%= link_to decidim.root_url(host: organization.host), "aria-label": t("front_page_link", scope: "decidim.accessibility", organization: organization.name) do %>
```

Comment:
Is it necessary to pass the organization variable here? Or is it enough with
```suggestion
  <%= link_to decidim.root_url(host: organization.host), "aria-label": t("front_page_link", scope: "decidim.accessibility") do %>
```
?

Reply:
Yes it is. Removed at 5223a37.